package.json

```diff
@@ -6,11 +6,15 @@
   "scripts": {
     "babel": "babel lib -d dist",
     "watch": "babel lib -d dist --watch",
-    "clean": "rm -f ./dist/*.js",
+    "clean": "rm -rf ./dist/*",
     "lint": "eslint lib/ test/ || exit 0",
     "lint:fix": "eslint --fix lib/ test/ || exit 0",
     "prepublish": "make babel",
-    "test": "mocha --reporter spec -r babel-core/register -r jsdom-global/register test/*.js"
+    "installPeerDeps": "./scripts/install-peer-deps.js",
```

Comment:
So this is necessary for the CI I'm guessing right? If we're hoping for this to be a reference for all future plugins to be built off of, it feels like a lot of extra overhead for every package to include something like this. Would there be a good way to keep this entirely in the circlie-ci environment maybe by having this is as a separately published package that is installed in the circle-ci container (I found one package already that does this but we can have our own too since it's not that complicated, https://www.npmjs.com/package/npm-install-peers). I feel like in order to make the CI and testing as easy to replicate across many packages, we should require as few files and configs in each project as possible.

Reply:
If you want to run the tests locally, you need a way to install locally

Reply:
so is every project going to need this then? If outside dependencies are needed it kind of feels more like integration tests than unit tests.

Reply:
These tests are unit tests that require peer dependencies to be installed due to required dependencies being included in the project as peer dependencies based on the way the application in structured

Reply:
Every project that has required deps listed as peer deps will need this script

Reply:
Yeah, I get what you're saying and I don't disagree. I'm just thinking of ways to make the testing framework and processes as portable as possible, and I think the best way to do that would be to come up with a system where all you need is to have a CI config that people can use in their own projects. One initial thought is that anything that is explicitly necessary for testing could be listed as a devDependency (it looks like only `HDPublicKey` from `bcoin` is needed for these tests right?). This is definitely not an urgent concern though, and something that we can just try and think about how to optimize moving forward.